### PR TITLE
🍭 The Sweet-o-meter by Anna Hasselström

### PR DESCRIPTION
**Pull request by annahasselstrom**
Link to my deployed project:

https://jovial-mcnulty-256bca.netlify.app/

My reflections on how this week's project turned out:

Disclaimer! To whomever is doing the code review - there are a lot of comments and I made them for me in order to try to understand the code. I realise this is not how it should look in the real world.

I've had a few weeks of feeling more and more insecure about learning to code, and actually, to some extent, stopped having fun when coding. This changed this week! And that made me very happy! This project has been fun to do. I like React. I like to think in components. It's fun!   🥳

As for levels, I have reached blue I think. My goal was to have one view/state per question but I'm not there. Yet.

Things I'd like to have clarified or explained in more detail:

Still struggling a bit on understanding the various directions data can flow and the relations between components and children/parents.

Also, what is a callback function?

### CODE REVIEW

File: code/src/components/Checkbox.js [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+          onChange={() => onLikesChange('kittens')} 
+        />
+        <span className="checkbox-label"></span>
+        <label htmlFor="likeStrawberries">Strawberries</label>
+        <input
+          id="likeStrawberries"
+          type="checkbox"
+          checked={userLikes.includes('strawberries')}
+          onChange={() => onLikesChange('strawberries')}
+        />
```
Comment (ClaireCaudwell) [inline code/src/components/Checkbox.js:26]: When I was using the keyboard to tab I found that I couldn't select either of the checkboxes using the arrows. I tried to do this by clicking enter when I had tabbed on to wither of the buttons, but this  submitted the form. Did you create custom checkboxes? Not sure what the solution would be, but I know that I also had issues selecting my checkboxes using the keyboard, as I had used customised checkboxes. Something the teacher can confirm what to do in this situation?

**Review by ClaireCaudwell — COMMENTED** [review]
Anna! I love your form. The design, theme and functionality works really well together. I can tell that you put a lot of time into this project and that you understand everything that you've done. 

The only comments are regarding the accessibility issues that I found, but to be honest I think you don't need to change these unless you really feel that you want to. And also if we get some feedback from the teacher checking the code review with suggestions on how these can be done!

I hope that you feel like you've achieved a lot, because you have 👌 well done 💥

File: code/src/components/Checkbox.js (hunk 1 above)
Comment (HIPPIEKICK) [reply code/src/components/Checkbox.js:26]: I think you should use the tab key to bring the cursor to the check-box and then hit the space bar on the keyboard to check or uncheck it. And Claire, I think the same goes for your radio buttons as we discussed - if you use tab and space they are selectable!

File: code/src/components/Radiobutton.js [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+            htmlFor="days" 
+            id="days"
+            key={days}
+            aria-label={days}>
+            <input className="input"
+              type="radio"
+              value={days}
+              onChange={(event) => setDaysPerWeek(event.target.value)}
+              checked={daysPerWeek === days}
+              />
```
[the remark this answers is not in the document]
Comment (HIPPIEKICK) [reply code/src/components/Radiobutton.js:28]: For me these are read out loud as well! @ClaireCaudwell I think you can mark this Pull request as approved 👍

[the remark this answers is not in the document]
Comment (ClaireCaudwell) [reply code/src/components/Radiobutton.js:28]: Yep will do. Thanks Matilda 👌

**Review by ClaireCaudwell — APPROVED** [review]

**Pull request closed without merging**